composer.json

```diff
@@ -31,7 +31,8 @@
         "orchestra/testbench": "^5.0 || ^6.0",
         "mockery/mockery": "^1.0",
         "phpunit/phpunit": "^9.1",
-        "friendsofphp/php-cs-fixer": "^2.16"
+        "friendsofphp/php-cs-fixer": "^2.16",
+        "guzzlehttp/guzzle": "^7.2"
```

Comment:
Could you remove `"guzzlehttp/guzzle": "^7.2"` we don't force any specific [http client](https://github.com/meilisearch/meilisearch-laravel-scout#install-the-http-client)

Reply:
I had to add this as some tests initialize an Http/Client and a PSR18 error would raise and let the command tests fail.

I'll try to fiddle around with mocking and see if I can make it run without the dependency. As it's a dev requirement, it wouldn't force any end users to install guzzle

Reply:
@shokme So we can either mock the `Meilisearch/Client::class` and count on the base that everything works fine, or we pull in a PSR18 compatible Client to check if everything works fine out of the box.

Reply:
I've just tested, it indeed does not work if we don't provide a PSR18 compatible client.
If I understand well the `guzzlehttp/guzzle` package will not be installed into the user projects because it's in the `require-dev` section of the `composer.json` file, right?
We can also mock the class, but I opened an issue about mocking #90

Reply:
@curquiza thanks for the fast reply.
Yes, the dependencies from the `require-dev` block is only installed in the local environment and is not pulled in on client integrations, as this is the point of this section. Same would be with the phpunit dependcy which is not installed on the integrators side. See https://getcomposer.org/doc/04-schema.md#require-dev